merlin-framework/src/main/java/gov/nasa/jpl/aerie/merlin/framework/annotations/ActivityType.java

```diff
@@ -100,4 +100,35 @@ enum Executor { Threaded, Replaying }
   @Retention(RetentionPolicy.CLASS)
   @Target({ ElementType.FIELD, ElementType.METHOD })
   @interface FixedDuration {}
+
+  /**
+   * Use when an activity's duration is indirectly determined only by its arguments.
```

Comment:
Should we include a pointer back to `@ControllableDuration` similar to on line 48?

Reply:
yes